modules/dkan/dkan_datastore/modules/dkan_datastore_api/dkan_datastore_api.module

```diff
@@ -360,7 +360,12 @@ function dkan_datastore_api_services_resources() {
 function _dkan_datastore_api_datastore_index() {
   if ($_SERVER['REQUEST_METHOD'] === 'GET') {
     $params = _dkan_datastore_api_get_params();
+    $cache_hash = md5(json_encode($params));
```

Comment:
@topicus can you document this function more completely? What kind of index is being created, and explain that this adds responses to the cache.